kythe/java/com/google/devtools/kythe/extractors/java/JavaCompilationUnitExtractor.java

```diff
@@ -318,7 +318,7 @@ public CompilationDescription extract(
    */
   private static List<String> removeDestDirOptions(Iterable<String> options) {
     return JavacOptionsUtils.removeOptions(
-        Lists.newArrayList(options), EnumSet.of(Option.S, Option.H));
+        Lists.newArrayList(options), EnumSet.of(Option.D, Option.S, Option.H));
```

Comment:
Add a TODO to add this back in (probably on the indexer side) to support modules.